07.ls_object/lib/display.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+require_relative 'file_status'
+
+class Display
+  COLUMN_COUNT = 3
+  SHORT_LIST_PADDING = 7
+  FILETYPES = {
+    'fifo' => 'p',
+    'characterSpecial' => 'c',
+    'directory' => 'd',
+    'blockSpecial' => 'b',
+    'file' => '_',
+    'link' => 'l',
+    'socket' => 's'
+  }.freeze
+  MODE_TABLE = {
```

Comment:
```suggestion
  SHORT_LIST_PADDING = 7
  
  FILETYPES = {
    'fifo' => 'p',
    'characterSpecial' => 'c',
    'directory' => 'd',
    'blockSpecial' => 'b',
    'file' => '_',
    'link' => 'l',
    'socket' => 's'
  }.freeze
  
  MODE_TABLE = {
```

みたいな感じで改行した方が読みやすいかも